CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 1.0.0
+* Removes unnecessary requirements for loading snapshots. (#433)
+* Adds snapshot loading capabilities. (#414)
+* Adds ability to change engine IO type. (#411) 
+* Adds ability to get Firecracker Version. (#410)
+* Deprecates pkg/errors. (#404)
```

Comment:
I think we can exclude this change from the changelog. A user probably wouldn't care.